backend/middleware/audit/audit.go

```diff
@@ -62,7 +66,11 @@ func (m *mid) UnaryInterceptor() grpc.UnaryServerInterceptor {
 		resp, err := handler(ctx, req)
 
 		if id != -1 {
```

Comment:
let's add a TODO and/or file an issue to move the response recording into a goroutine so it's async